jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonUnmarshaller.java

```diff
@@ -2911,29 +2954,30 @@ private void createSubProcessDiagram(BPMNPlane plane, FlowElement flowElement, B
                     subProcessFlowElement.getAnyAttribute().add(extensionEntry);
                 }
 			} else if (subProcessFlowElement instanceof SequenceFlow) {
-				SequenceFlow sequenceFlow = (SequenceFlow) subProcessFlowElement;
-				BPMNEdge edge = factory.createBPMNEdge();
-				edge.setBpmnElement(subProcessFlowElement);
-				DcFactory dcFactory = DcFactory.eINSTANCE;
-				Point point = dcFactory.createPoint();
-				if(sequenceFlow.getSourceRef() != null) {
-					Bounds sourceBounds = _bounds.get(sequenceFlow.getSourceRef().getId());
-					point.setX(sourceBounds.getX() + (sourceBounds.getWidth()/2));
-					point.setY(sourceBounds.getY() + (sourceBounds.getHeight()/2));
-				}
-				edge.getWaypoint().add(point);
-				List<Point> dockers = _dockers.get(sequenceFlow.getId());
-				for (int i = 1; i < dockers.size() - 1; i++) {
-					edge.getWaypoint().add(dockers.get(i));
-				}
-				point = dcFactory.createPoint();
-				if(sequenceFlow.getTargetRef() != null) {
-					Bounds targetBounds = _bounds.get(sequenceFlow.getTargetRef().getId());
-					point.setX(targetBounds.getX() + (targetBounds.getWidth()/2));
-					point.setY(targetBounds.getY() + (targetBounds.getHeight()/2));
-				}
-				edge.getWaypoint().add(point);
-				plane.getPlaneElement().add(edge);
+                SequenceFlow sequenceFlow = (SequenceFlow) subProcessFlowElement;
```

Comment:
For better readability I propose to align this block of code in standard way:
`
if (...) {
    // any code here
} else if (...) {
   // any code here
}
`